src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import duke.views.cli.Cli;
+import duke.views.cli.strategies.MultiType;
+import duke.views.gui.Launcher;
+import javafx.application.Application;
```

Comment:
Would it be better to move this import line above the other imports instead? 😬 Since you've put your java imports above your duke imports in your other classes. (Don't forget to add a blank line in between to keep the formatting consistent :D)

Reply:
👍

Reply:
Strangely, this is the way Checkstyle wants it. 🤔